src/main/java/org/zalando/nakadi/repository/kafka/KafkaTopicRepository.java

```diff
@@ -177,6 +177,7 @@ public boolean partitionExists(final String topic, final String partition) throw
 
             producer.send(kafkaRecord, ((metadata, exception) -> {
                 if (null != exception) {
+                    LOG.warn("Failed to publish to kafka topic {}", topicId, exception);
```

Comment:
@rcillo It seems that I lost initial comment.
My point is that for now it's just a warning, because it may be normal system behavior (for example transferring leadership).
In case if warning is present for a long time, than it's an error.